src/styles/main.scss

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$day-size: 100px;
+$gap: 1px;
+
+.calendar {
+  max-width: calc($day-size * 7 + $gap * 6);
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  &__day {
+    background-color: #eee;
+    position: relative;
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      cursor: pointer;
+      transition-duration: 0.5s;
```

Comment:
Consider adding `transition-property` to specify which properties should transition. This can improve performance by limiting the properties that are animated.